expo/expo
# [web] Removed all config in favor of an identical universal solution

# Why

Reduce all of the babel overhead.

Related: 
- https://github.com/expo/expo/pull/4973
- https://github.com/expo/expo/pull/4889

# How

Given that tree-shaking is broken in all of the latest babel, we'll pull out the custom plugin in favor of metro with the CJS disabled.

# Test Plan

- Adding tests to babel-preset-expo
- Run `expo start --web-only --web` in NCL
- Copy this config into a brand new project outside of expo and ensure that it runs with the latest CLI.
  - Create new project
  - Copy the contents of `packages/babel-preset-expo/index.js` into `node_modules/babel-preset-expo/index.js`
  - Run `expo start --web-only --web`
- Build a project
  - Create new project
  - Copy the contents of `packages/babel-preset-expo/index.js` into `node_modules/babel-preset-expo/index.js`
  - Add reporting to the project
  - Run `expo build:web`
  - Reports should show a low package binary size
